test/e2e/run_test.go

```diff
@@ -282,13 +301,16 @@ var _ = Describe("Podman run", func() {
 			"--rootfs", rootfs + ":O", "cat", testFilePath})
 		osession.WaitWithDefaultTimeout()
 		Expect(osession).Should(ExitCleanly())
+		Expect(osession.OutputToString()).To(Equal(uniqueString))
 
 		// Test podman start stop with overlay
 		osession = podmanTest.Podman([]string{"run", "--name", "overlay-foo", "--security-opt", "label=disable",
 			"--rootfs", rootfs + ":O", "echo", "hello"})
 		osession.WaitWithDefaultTimeout()
 		Expect(osession).Should(ExitCleanly())
+		Expect(osession.OutputToString()).To(Equal("hello"))
 
+		// FIXME, this seems pointless??????????
```

Comment:
Special attention here please

Reply:
start stop needs special testing, i.e. missing cleanup from previous run might cause failures if you try to start again.